### ISSUE TITLE
Lensfun DB missing from installer

### ISSUE THREAD

**Issue by Beep6581** [aferrero2707/rt-win64#2]
From the job 216.4 log:
```
Reading http://lensfun.sourceforge.net/db/versions.json â€¦
Reading http://wilson.bronger.org/lensfun-db/versions.json â€¦
Successfully updated the database in /var/lib/lensfun-updates/version_2.
cp: cannot stat '/var/lib/lensfun-updates/version_1/*': No such file or directory
```
https://travis-ci.org/aferrero2707/rt-win64/jobs/645165878

**Comment by Beep6581** [comment]
BTW @aferrero2707 I'm still using DB `version_1` on my machine, so not sure what changes `version_2` entails other than the dir name.

**Comment by Beep6581** [comment]
Fixed by @aferrero2707  in 4b861d9
